### PR TITLE
Release Notes7.8 (updated)

### PR DESCRIPTION
**Pull request by fpetru**
- bring a small overview section and more pictures, highlighting different features & SBCs supported

### CODE REVIEW

File: docs/hardware.md [hunk 1]
```diff
@@ -23,13 +23,9 @@ DietPi runs also on one of the first models **Raspberry Pi 1**, launched in 2012
 
 ![Raspberry Pi 1 Model B photo](assets/images/raspberry-pi-1b.jpg){: width="500" height="242" loading="lazy"}
 
-and even on the compact form **Raspberry Pi Zero W**
+and even on the compact form **Raspberry Pi Zero**, both **Zero W / WH** and **Zero 2 W**
 
-![Raspberry Pi Zero W photo](assets/images/raspberry-pi-zero-w.jpg){: width="500" height="333" loading="lazy"}
```
Comment (MichaIng) [inline docs/hardware.md:28]: Don't forget to delete the image, when it's not used anymore.

File: docs/hardware.md [hunk 2]
```diff
@@ -23,13 +23,9 @@ DietPi runs also on one of the first models **Raspberry Pi 1**, launched in 2012
...
-
-Quick intro:
-
-<iframe src="https://www.youtube-nocookie.com/embed/sajBySPeYH0?rel=0" frameborder="0" allow="fullscreen" width="560" height="315" loading="lazy"></iframe>
```
Comment (MichaIng) [inline docs/hardware.md:32]: Was it intended that the YouTube video got removed?

File: docs/hardware.md [hunk 3]
```diff
@@ -68,6 +64,7 @@ Quick intro:
                             <li>Raspberry Pi 4 Model B<br>Compute Module 4</li>
                             <li>Raspberry Pi 3 Model A+ / B / B+<br>Compute Module 3 / 3+</li>
                             <li>Raspberry Pi 2 Model B v1.1 / B v1.2</li>
+                            <li>Raspberry Pi Zero 2 W</li>
```
Comment (MichaIng) [inline docs/hardware.md:67]: Doubled, it's already at the top of the list 🙂.

File: docs/releases.md [hunk 4]
```diff
@@ -4,28 +4,66 @@
...
-- [**DietPi-Drive_Manager**](../dietpi_tools/#dietpi-drive-manager) :octicons-arrow-right-16: On ext4 filesystems, the reserved blocks percentage is now shown and can be changed as well if the drive is not currently mounted.
-- [**DietPi-RAMlog**](../software/log_system/#dietpi-ramlog) :octicons-arrow-right-16: The /var/log directory structure is now synced to the persistent disk storage after software installs and via daily cron job to prevent missing logs files or directories in case of unclean shutdowns, which usually lead to failing service startups.
+- [**DietPi-Backup**](../dietpi_tools/#dietpi-backup-backuprestore)
+    - A feature has been added to enable daily system backup via cron job. Many thanks to @Lycidias93 for suggesting this feature: <https://github.com/MichaIng/DietPi/issues/3871>
+        ![DietPi Backup Daily](assets/images/dietpi-backup-daily.jpg){: width="713" height="300" loading="lazy"}
```
Comment (MichaIng) [inline docs/releases.md:48]: No need to change, but for the future: Such console screens look much better as PNG and also the file size is smaller. JPEG seems to not deal well with material like sharp edges and non-fluent sharp colour cuts, while it's better for natural photos with many colours, organic colour gradients and contours.

File: docs/hardware.md (hunk 3 above)
Comment (fpetru) [reply docs/hardware.md:67]: True. Removed, just wanted to be sure... :)

File: docs/hardware.md (hunk 2 above)
Comment (fpetru) [reply docs/hardware.md:32]: Maybe I was quick on removing it - my idea behind is to not promote it too much - with the launch of `Zero 2 W`, the RP4 is not the latest.

File: docs/hardware.md (hunk 1 above)
Comment (fpetru) [reply docs/hardware.md:28]: done

File: docs/releases.md (hunk 4 above)
Comment (fpetru) [reply docs/releases.md:48]: good idea. changed

File: docs/hardware.md (hunk 2 above)
Comment (fpetru) [reply docs/hardware.md:32]: Replaced the video, highlighting the `New Zero`

Comment (MichaIng) [reply docs/hardware.md:32]: Good idea 👍

**Review by MichaIng — APPROVED** [review]

**Comment by MichaIng** [comment]
Looks good to me. Ready from your end?

**Pull request merged**